EdFi.Buzz.Database/migrations/sqls/20201117102215-create-attendance-tables-up.sql

```diff
@@ -0,0 +1,17 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TABLE buzz.Attendance (
+    StudentSchoolKey VARCHAR(45) NOT NULL,
+    StudentKey VARCHAR(32) NOT NULL,
+    SchoolKey VARCHAR(30) NOT NULL,
+    ReportedAsPresentAtSchool NUMERIC(5,2) NOT NULL DEFAULT 0,
+    ReportedAsAbsentFromSchool NUMERIC(5,2) NOT NULL DEFAULT 0,
+    ReportedAsAbsentFromSchoolReportedAsPresentAtHomeRoom NUMERIC(5,2) NOT NULL DEFAULT 0,
```

Comment:
Can you check in AMT script to see what the correct name should be here and change it? We already have a bug ticket created to fix it in AMT.